images/simple-image.mk

```diff
@@ -54,4 +54,4 @@ push_latest: confirm-master build
 	docker tag $(IMG):$(TAG) $(IMG):latest
 	docker push $(IMG):latest
 
-push: push_versioned push_latest
+push:: push_versioned push_latest
```

Comment:
Curious why this change

Reply:
Needed so push:: in the prow-tests/Makefile can add a requirement.